lib/update_app_name.rb

```diff
@@ -0,0 +1,58 @@
+# frozen_string_literal: true
+
+# Service for updating Heroku review app's name, to keep app name convension: 'dahlia-lap-full-pr-XXX'.
+class UdpateAppName
+  SKIPPED_APPS = %w[dahlia-lap-full dahlia-lap-nonepic dahlia-lap-qa dahlia-lap-production dahlia-lap-preprod].freeze
```

Comment:
this scares me a little bit, to be renaming all of them instead of renaming just the one that's being created. What's the reasoning behind going through all apps instead of the app that's being deployed?

I wouldn't want to accidentally affect a different project too. I know it's filtered by dahlia-lap but still seems a bit sketchy

Reply:
Sure, I understand your concerns. Our first approach was to load review apps via pipeline. But review apps endpoint doesn't return app names. And there is no filtering in the Heroku API: https://devcenter.heroku.com/articles/platform-api-reference#app-list 
But maybe a safer filtering approach would be to get all there review apps via pipeline, then all apps, and continue filtering by ID, rather then name? Well first via ID then name to get the incorrect apps. 
Chris was initially in contact with heroku support and thats the way they suggested. Ideally we would rather get ID of the app in question during the deployment, but I dont think its possible.

Reply:
yeah that would make me more comfortable. Are you up for trying that out?

Reply:
Sure, will do